docs/annexes/using-SPDX-to-comply-with-industry-guidance.md

```diff
@@ -0,0 +1,50 @@
+# Annex F Using SPDX to comply with Norms, Standards and Regulation (Informative)
```

Comment:
@karsten-klein is there a reason you called this Annex F? Seems a bit arbitrary since each annex is it's own separate markdown file in the repo.